### DISCUSSION TITLE
Volume type sc1 on eks

### DISCUSSION THREAD

**Discussion by sunilemanjee** [openebs/openebs#3612] [Q&A] [answered]
I'm using helm operator for openebs on eks and it works great. However, I want to use a different type of EBS volume on eks, volume type sc1.  https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html#EBSVolumeTypes_sc1.  yes I know it is slow but its for a specific use case where I need slow disk for my containers.

I don't see any docs where specify the volume type

For example, here is what I would do with a yaml
```cat << EoF > ${HOME}/environment/ebs_statefulset/mysql-storageclass.yaml
kind: StorageClass
apiVersion: storage.k8s.io/v1
metadata:
  name: mysql-gp2
provisioner: ebs.csi.aws.com # Amazon EBS CSI driver
parameters:
  type: sc1
volumeBindingMode: WaitForFirstConsumer # EBS volumes are AZ specific
reclaimPolicy: Delete
mountOptions:
- debug
EoF```

Any guidance appreciated.

**Comment by avishnu** [comment, marked as answer]
This is likely an EKS configuration, and not related to OpenEBS. Please refer to the EKS documentation.
